docs/reference/commandline/volume_create.md

```diff
@@ -21,10 +21,12 @@ parent = "smn_cli"
 
 Creates a new volume that containers can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
 
+    ```bash
     $ docker volume create --name hello
     hello
 
     $ docker run -d -v hello:/world busybox ls /world
+    ```
```

Comment:
Oh, sorry, no, you need to use either indentation (4 spaces) *or* GitHub style fences. GitHub style fences are nice, because we can control what kind of code-highlighting should be used.

Because you now have both indented *and* added fences, it becomes:

<img width="473" alt="screen shot 2016-03-02 at 11 01 51" src="https://cloud.githubusercontent.com/assets/1804568/13457064/61081c4c-e066-11e5-924b-7bd04bf9b662.png">